File: minimumAbsoluteDifference_hacker_rank.js
function minimumAbsoluteDifference(arr) {
    let min = [0, 1];
    let i = 0;
    while (i < arr.length - 1) {
        let j = i + 1;
        let num1 = arr[i];
        while (j < arr.length) {
            let num2 = arr[j];
            if (Math.abs(num1 - num2) < Math.abs(arr[min[0]] - arr[min[1]])) {
                min = [i, j];
            }
            j++
        }
        i++
    }

    return Math.abs(arr[min[0]] - arr[min[1]]);
}

File: minimumAbsoluteDifference_hacker_rank.rb
def minimumAbsoluteDifference(arr)
    min = [0,1];
    i = 0;
    while i < arr.length-1
        j = i+1
        num1 = arr[i]
        while j < arr.length
            num2 = arr[j]
            number = (num1 -num2).abs
            min_number = (arr[min[0]] - arr[min[1]]).abs
            if(number < min_number)
                min = [i, j]
            end
            j += 1
        end
        i+=1
    end
    return (arr[min[0]] - arr[min[1]]).abs
end